adminservice/src/intTest/groovy/nu/yona/server/InitialGoalCreationTest.groovy

```diff
@@ -1,6 +1,7 @@
 package nu.yona.server
 
 import groovyx.net.http.RESTClient
+import groovyx.net.http.HttpResponseException
```

Comment:
Ah, I did not know that you intended to remove the initial functionality of
add/remove goals from the adminservice. In that case I can indeed remove a
lot of code. Do you want to remove the entire GoalController, or leave only
the method getAllGoals() there? Do you want to remove the public
add/delete/update methods from the GoalService too?

On Tue, Nov 17, 2015 at 3:24 PM, Bert Roos <notifications@github.com> wrote:

> In
> adminservice/src/intTest/groovy/nu/yona/server/InitialGoalCreationTest.groovy
> <https://github.com/yonadev/yona-server/pull/7#discussion_r45064892>:
>
> > @@ -1,6 +1,7 @@
> >  package nu.yona.server
> >
> >  import groovyx.net.http.RESTClient
> > +import groovyx.net.http.HttpResponseException
>
> This test might go away. The initialization work is done through your new
> code. The testing work needs to happen against the /goals/ service in the
> appservice. The update feature will disappear, as we now have the file to
> load the goals.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yonadev/yona-server/pull/7/files#r45064892>.
>